typo-fix-gdpr-page (#327)
ScilifelabDataCentre/RDM-guidelines: content/topics/gdpr-ethical-review-glossary.md

*In Swedish:* registrerad

> Any person who can be identified, directly or indirectly, via an identifier such as a name, an ID number, location data, or via factors specific to the person's physical, physiological, genetic, mental, economic, cultural or social identity.

Reviewer: ```suggestion
Any person that can be identified, directly or indirectly, via an identifier. Identifiers may include a name, an ID number, location data, or factors specific to the person's physical, physiological, genetic, mental, economic, cultural, or social identity.
```

Author: I do not really like to close a sentence and begin the next one with the same word (in this case identifier). But it can work as you have suggested!

-   https://www.imy.se/en/organisations/data-protection/this-applies-accordning-to-gdpr/information-security/

> ### **lawful ground, legal basis**

Reviewer: ```suggestion
### **lawful grounds, legal basis**
```

Author: I agree with you that "Lawful grounds" is the most appropriate!
